distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -134,12 +138,15 @@ type Column
                 constant ->
                     SQL_Expression.Constant constant
 
+            self_maybe = if include_self then [self] else []
+            self_expression_maybe = if include_self then [self.expression] else []
```

Comment:
It seems that `include_self=False` is not used anywhere anymore, right?
Then I think we could undo this to simplify `make_op`, as it does not seem very likely we will have more ops using it.

Reply:
Done.